src/base-input.js

```diff
@@ -440,6 +442,15 @@ export default class BaseInput extends LitElement {
 
     this.isValid = this.noValidate ? true : this.inputElement.checkValidity();
     this.internalError = this.isValid ? null : this.inputElement.validationMessage;
+
+    this.dispatchEvent(new CustomEvent('auroInput__validated', {
```

Comment:
This feels like an awkward name for an event. Do we have standards established around this?

Personally, I'd prefer something like `auro-validated`. `auroInput` is redundant since you'll listen to this event on the auro-input element, and the `__` isn't something you normally see in event names.

Reply:
> This feels like an awkward name for an event.

We did discuss this, but we never landed on a specific standard. The goal is to clearly expose that the events are coming from Auro and not to be confused with other events native or otherwise. 

The question here is, do we need to also discerne between elements as well? E.g. `auro-` or specifically `auroInput-`?

What about `auro-inputValidated`? There we can clearly see that it's an Auro event and the event is validating the input. 

> and the `__` isn't something you normally

I agree.

Reply:
This should be [documented](https://auro.alaskaair.com/generator/best-practice) when a decision is made.